src/App.js

```diff
@@ -75,6 +86,25 @@ export default class App extends Component {
 
   render() {
     const totalCountString = this.state.totalCount > 0 ? `Total alternatives: ${this.state.totalCount}` : ''
+    const loginSection = this.state.authed ? (
```

Comment:
Since your saving this in the render method, wouldn't it makes sense to move this button into its own component that takes this.state.authed as a prop.

Reply:
Could in the next iteration I'm hooking up https://aws.amazon.com/cognito/ so this will get refactored.